src/main/java/org/jabref/gui/util/ValueTableCellFactory.java

```diff
@@ -122,9 +123,11 @@ protected void updateItem(T item, boolean empty) {
                         if (StringUtil.isNotBlank(tooltipText)) {
                             Screen currentScreen = Screen.getPrimary();
                             double maxWidth = currentScreen.getBounds().getWidth();
+                            setTextOverrun(OverrunStyle.LEADING_ELLIPSIS);
```

Comment:
Changing this will have major impact all over JabRef, this needs comprehensive testing in the ui

Reply:
Yes you are right, because it is a shared file. I tried to fix it, it should be correct now.